ironic/drivers/modules/drac/raid.py

```diff
@@ -884,6 +972,65 @@ def _check_node_raid_jobs(self, task):
                 self._clear_raid_config_job_failure(node)
```

Comment:
This block of code, starting with "if not node.driver_internal_info['raid_config_job_ids']:" needs to be combined with the the if statement below on line 975.  As it is, when the config job finishes, the above code will call _resume_cleaning(), which is not what we want if there is another substep to execute.

Reply:
done.